### DISCUSSION TITLE
Is Tasmota powered RGB light bulb useful as ambilight?

### DISCUSSION THREAD

**Discussion by DevelopMan303** [arendst/Tasmota#20421] [Show and tell]
Hi Fellas,
Today is a snowy day and I have been playing around to see if a Tasmota powered RGB bulb could be used as an ambilight. It turned out that the proof of concept worked almost on the first try. 

Implementation: I run a python script which captures a screen shot from the left side and then sends this color after some filtering to the Tasmota bulb via http requests. 

The big (and maybe only) advantage, is that it can be used without any additional setup/wires and so on. 

I am wondering, if I should evolve this in some way. 
So I have the following questions: Do you have any use for this and if so, how (e.g. as a Kodi plugin)?

Best regards 

Beware: My camera sees the color very differently (especially blue) than myself.  

![POC_ mp4](https://github.com/arendst/Tasmota/assets/54041681/6370753a-67ad-4ede-936f-0cf3b0b0ac20)

**Comment by sfromis** [comment]
Looking forward, one way of integrating live color control (likely with lower latency than processing commands) would be to use the optional DDP feature https://github.com/arendst/Tasmota/pull/14017 - also without additional setup.

Of course, using the Distributed Display Protocol would mean a different way of coding, but does open a door to many integration options, in addition to the lower latency you can expect. Maybe there also are existing integrations with a camera and DDP. My guess would be that Kodi and DDP can be brought to talk.
